### PR TITLE
Merge from dependabotprs

### PR DESCRIPTION
**Pull request by jbum**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,57 +10,55 @@
     "@cagov/lookup": "^1.0.3",
     "@cagov/pagerating": "^2.0.0",
     "@cagov/step-list": "^1.0.11",
-    "@webcomponents/webcomponentsjs": "^2.5.0",
+    "@webcomponents/webcomponentsjs": "^2.6.0",
     "awesomplete": "^1.1.5",
     "awesomplete-es6": "^1.2.2",
     "element-closest-polyfill": "^1.0.4",
-    "jest": "^27.0.6",
+    "jest": "^27.3.1",
```
Comment (aaronhans) [inline package.json:17]: This shouldn't be necessary anymore

**Review by aaronhans — APPROVED** [review]
lot of build tool updates, thanks Jim!

**Comment by aaronhans** [comment]
Looks like axe-playwright is required, not sure why it is missing but that is used in the playwright test script and caused that build validation to fail

**Comment by jbum** [comment]
Cool fixing dependabotprs...

**Pull request merged**